.env.example

```
@@ -1,20 +1,22 @@
-# API Keys (Required)
-ANTHROPIC_API_KEY=your_anthropic_api_key_here  # Format: sk-ant-api03-...
-PERPLEXITY_API_KEY=your_perplexity_api_key_here  # Format: pplx-...
+# API Keys (Required for using in any role i.e. main/research/fallback -- see `task-master models`)
+ANTHROPIC_API_KEY=YOUR_ANTHROPIC_KEY_HERE
+PERPLEXITY_API_KEY=YOUR_PERPLEXITY_KEY_HERE
+OPENAI_API_KEY=YOUR_OPENAI_KEY_HERE
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENAI_API_KEY key should go before the PERPLEXITY_API_KEY key

(UnorderedKey)